ekncontent/ekncontent/eknc-article-object-model.c

```diff
@@ -321,6 +347,23 @@ eknc_article_object_model_get_authors (EkncArticleObjectModel *self)
 }
 
 /**
+ * eknc_article_object_model_get_temporal_coverage:
+ * @self: the model
+ *
+ * Get the temporal coverage over the article.
+ *
+ * Returns: (transfer none): the resources GVariant
```

Comment:
We haven't been doing this conscientiously yet, but this should probably have a `Since: 2` annotation.